uktrade/data-hub-frontend
# feature/CLS2-375-reactify-edit-one-list

## Description of change

Move the logic for the edit one list page from a nunjucks controller into a React component. The tasks and form have been copied without modification, and a new Task used to get the one list information that was previously loaded in the controller

The existing controller and nunjucks have all been removed

## Test instructions

Use any exiting company, for example http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/edit-one-list, you should be able to edit and make changes to the one list team in the same way as you can on the dev site

## Screenshots

![chrome_izNXhWYXl1](https://github.com/uktrade/data-hub-frontend/assets/102232401/79ec8187-0059-4f66-9718-a5f098ec1fe6)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
